src/battle_anim.c

```diff
@@ -793,6 +793,10 @@ static void Cmd_end(void)
 
     if (!continuousAnim) // May have been used for debug?
     {
+        // Debugging - ensure no hanging mon bg tasks
+        if (FuncIsActiveTask(Task_UpdateMonBg))
+            DebugPrintfLevel("Move %d animation still has Task_UpdateMonBg active at the end!", gAnimMoveIndex);
```

Comment:
I think `DebugPrintfLevel` takes `level` as its first argument. Maybe you want to pass something like `MGBA_LOG_ERROR` for that? Or you could use plain `DebugPrintf` and skip the level argument entirely.

Reply:
good call. done